Lauhdutin/@Resources/Frontend/GUI.lua

```diff
@@ -264,8 +264,58 @@ end
 			return
 		end
 		local tResult = {}
+		local HiddenFilter = SKIN:GetVariable('HiddenFilters')
+		--HiddenFilter = Enabled or Disabled
+		--if HiddenFilter == "Enabled" then
 		if StartsWith(asPattern, 'steam:') then
-			return FilterByTag(atTable, asPattern, 'steam', GAME_KEYS.PLATFORM, PLATFORM.STEAM), true
+			asPattern = asPattern:sub(7)
+			if StartsWith(asPattern, 'i') then
+			for i, game in ipairs(T_ALL_GAMES) do
+				if game["platform"] == 0 then
```

Comment:
Fields of the `GAME_KEYS` table should be used as the key instead. If the value of that field needs to be changed in the future, then every instance of `"platform"` would have to be manually updated as it currently is. Using `GAME_KEYS.PLATFORM` as the key means that only one line of code needs to be updated, if the value needs to be changed in the future.

The same applies to `== 0`, which should be replaced with `== PLATFORM.STEAM`.